plugins/bufbuild/connect-go/v1.10.0/buf.plugin.yaml

```diff
@@ -0,0 +1,19 @@
+version: v1
+name: buf.build/bufbuild/connect-go
+plugin_version: v1.10.0
```

Comment:
Might be a good time to discuss how we're going to handle this repository and BSR references wrt the connectrpc move.